src/content/docs/mobile-apps/android.mdx

```diff
@@ -40,17 +35,27 @@ The user interface for New Relic's Android app provides functionality similar to
   Depending on your New Relic account, you may need to [further authenticate your user](/docs/mobile-apps/new-relic-mobile-apps/authentication-alerts/user-settings-authentication).
 </Callout>
 
-## Features [#features]
-
-The New Relic Android app includes data about these features:
+<SideBySide>
+  <Side>
+    ## New Relic capabilities [#features]
 
-* APM metrics, both real-time and [historical](#time) data, including health maps. Select the transaction <Icon name="fe-repeat"/>
-  icon to see detailed transaction metrics, or an **Overview** chart to view summary charts of your top five transactions. Select the <Icon name="fe-filter"/>
-  icon to filter by [labels and categories](/docs/data-analysis/user-interface-functions/labels-categories-organize-your-apps-servers).
-* Browser monitoring metrics, including average page load time, Apdex, average throughput, and more.
-* Infrastructure monitoring.
-* Mobile monitoring, including [crash reports](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-dashboards/mobile-apps-crash-dashboard), network errors, API calls, and active user count.
-* Event notifications, including [mobile alerts](/docs/mobile-apps/new-relic-ios-app/features/ios-alerting) wherever you are, plus [deployment notifications](/docs/applications-menu/deployments-dashboard) and [notes](/docs/site/public-and-private-notes).
+    * **Alerts**: View current and past incidents, acknowledge incidents, and tap through to see performance data for related entities.
+    * **APM**: See real-time and historical data across transactions, errors, databases, external services, and more.
+    * **Browser monitoring**: Track throughput, load times, errors, AJAX requests and more across all your front-end web apps.
+    * **Dashboards and NRQL**: View your dashboards, or dive deeper with NRQL. You can build custom NRQL queries, or you can view the NRQL for any chart by tapping on the chart, then tapping the overflow <Icon name="fe-more-horizontal" /> icon and selecting **View query**.
+    * **Explorer**: Browse everything else in New Relic, from AWS services to network performance.
+    * **Infrastructure monitoring**: See the health of your hosts.
```

Comment:
Seems weak compared to the rest of the bullets. Yoinking something from the intro to infra doc could help:

`**Infrastructure monitoring**: See the health of your hosts running in the cloud, on dedicated hosts, or containers running in orchestrated environments.`